java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/GenericMqttFactory.java

```diff
@@ -0,0 +1,23 @@
+package com.cumulocity.generic.mqtt.client;
+
+/**
```

Comment:
nitpick: Links should not link back to the same class. I'd suggest/prefer either:
 - <code>GenericMqttFactory</code> is used to configure and create instances of...
 Or better still
 - Configures and create instances of... 
   (That is, current class is implicit)

Reply:
Not relevant after recent changes.